hypervisor/qemu/qemu.go

```diff
@@ -35,17 +34,6 @@ func qemuContext(ctx *hypervisor.VmContext) *QemuContext {
 	return ctx.DCtx.(*QemuContext)
 }
 
-func InitDriver() *QemuDriver {
-	cmd, err := exec.LookPath("qemu-system-x86_64")
```

Comment:
could you define multiple QEMU_SYSTEM_EXE in different arch-specific files
rather than define multiple InitDriver() in arch-specific files?